ibis/expr/datatypes.py

```diff
@@ -613,6 +613,46 @@ def __init__(
         self.geotype = geotype
         self.srid = srid
 
+    def __str__(self) -> str:
+        _geo_op = self.name.lower()
```

Comment:
Any particular reason to use an underscore in the variable name?

Reply:
nop. I will remove the `_` here.